### ISSUE TITLE
[Feature Request] Exclude/Blacklist to avoid conflict on some websites

### ISSUE THREAD

**Issue by lemonadeforlife** [aynorway/web-video-speed-controller#1]
Since this extension has memory option to remember playback speed, we often might end up overriding other websites as well. This feature might come really handy in this situation.

**Comment by aynorway** [comment]
Absolutely, your suggestion makes a lot of sense and I truly appreciate it. I'm convinced it's a valuable addition and I'll definitely incorporate it. Many thanks for your insight! I'll make sure to roll out this update as soon as I find the time.
